demisto_sdk/commands/run_unit_tests/unit_tests_runner.py

```diff
@@ -192,19 +199,16 @@ def unit_test_runner(file_paths: List[Path], verbose: bool = False) -> int:
                         )
                     exit_code = 1
                 else:
-                    logger.info(f"All tests passed for {filename} in {docker_image}")
+                    logger.info(
+                        f"All tests passed for {integration_script.path} in {docker_image}"
```

Comment:
do we really need the docker image in this log?

Reply:
Yes, as in the future it will run in native docker image as well, and it could be helpful to tell on which docker image it was run exactly.